src/components/UpgradeStatus.tsx

```diff
@@ -46,14 +45,10 @@ const UpgradeStatus: React.FC<IProps> = ({
       width="100%"
     >
       <Heading fontSize="sm" fontWeight="normal" textTransform="uppercase">
-        {/* TODO: Re-enable after i18n implemented */}
-        {/* <Translation id="consensus-when-shipping" /> */}
-        When's it shipping?
+        {t("common:consensus-when-shipping")}
```

Comment:
Is `common:` necessary? aren't we loading `common` by default?

Reply:
Needed to be specified as a prefix when another namespace is being used too, I'll add this to the docs